docs/standard/library-guidance/cross-platform-targeting.md

```diff
@@ -0,0 +1,93 @@
+
+> [!TIP]
+> The .NET team [offers a Roslyn analyzer](../analyzers/api-analyzer.md) to help you discover possible issues.
+
+**✔️ DO** start with including a `netstandard2.0` target.
+
+> Most general-purpose libraries should not need APIs outside of .NET Standard 2.0. .NET Standard 2.0 is supported by all modern platforms and is the recommended way to support multiple platforms with one target.
+
+**❌ AVOID** including a `netstandard1.x` target.
+
+> .NET Standard 1.x is distributed as a granular set of NuGet packages, which creates a large package dependency graph and results in developers downloading a lot of packages when building. Modern .NET platforms, including .NET Framework 4.6.1, UWP and Xamarin, all support .NET Standard 2.0. You should only target .NET Standard 1.x if you specifically need to target an older platform.
+
+**✔️ DO** include a `netstandard2.0` target if you require a `netstandard1.x` target.
+
+> All platforms supporting .NET Standard 2.0 will use the `netstandard2.0` target and benefit from having a smaller package graph while older platforms will still work and fall back to using the `netstandard1.x` target.
+
+**❌ DO NOT** include a .NET Standard target if the library relies on a platform-specific app model.
+
+> For example, a UWP control toolkit library depends on an app model that is only available on UWP. App model specific APIs will not be available in .NET Standard.
+
+## Multi-targeting
+
+Sometimes you need to access framework-specific APIs from your libraries. The best way to call framework-specific APIs is using multi-targeting, which builds your project for many [.NET target frameworks](../frameworks.md) rather than for just one.
+
+To shield your consumers from having to build for individual frameworks, you should strive to have a .NET Standard output plus one or more framework-specific outputs. With multi-targeting, all assemblies are packaged inside a single NuGet package. Consumers can then reference the same package and NuGet will pick the appropriate implementation. Your .NET Standard library serves as the fallback library that is used everywhere, except for the cases where your NuGet package offers a framework-specific implementation. Multi-targeting allows you to use conditional compilation in your code and call framework-specific APIs.
+
+![NuGet package with multiple assemblies](./media/cross-platform-targeting/nuget-package-multiple-assemblies.png "NuGet package with multiple assemblies")
+
+**✔️ CONSIDER** targeting .NET implementations in addition to .NET Standard.
+
+> Targeting .NET implementations allows you to call platform-specific APIs that are outside of .NET Standard.
+>
+> Do not drop support for .NET Standard when you do this. Instead, throw from the implementation and offer capability APIs. This way, your library can be used anywhere and supports runtime light-up of features.
+
+**❌ DO NOT** use multi-targeting with .NET Standard if your source code is the same for all targets.
```

Comment:
Should this be an AVOID as it contradicts both the suggestion above and the suggestion below?

Reply:
True, there are some rare valid reasons to do this.